sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/implementation/models/package-info.java

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+/** Package containing the data models for CommunicationIdentityClient. Azure Communication Identity Service. */
```

Comment:
The administration package should not have references to CommunicationIdentityClient anymore. Let's make sure this is removed.

Reply:
Deleted the models directory since there is no file left